campus-cyberinfrastructure.md

```diff
@@ -59,8 +59,8 @@ The OSG team actively works with campuses to integrate them smoothly into the OS
 
 Many other campuses and other sites, in addition to CC* awardees, contribute resources to the Open Science Pool. <a href="https://map.opensciencegrid.org/map/iframe?view=EduvsOther#29.22889,-90.08789|4|terrain" target="_blank">View them here.</a> CC* awardees are a subset of the overall contributions by educational institutions. In fact, many campuses have used their Open Science Pool contributions to aid in the success of a subsequent CC* award application to NSF. 
 
-<h2>Open Science is Open to You!</h2>
-These Open Science Pool is a <a href="https://research.cs.wisc.edu/htcondor/" target="_blank">HTCondor pool</a> that OSG users access via 'access points' operated by the OSG or by campuses supporting 'local' access to OSG. Any researcher at a US academic, government, or non-profit institution can can use the Open Science Pool via OSG-operated access points to harness the capacity of the Open Science Pool by going to <a href="https://www.osgconnect.net/" target="_blank">OSG Connect and get an OSG Account</a>. (It is free!) The researcher does not have to belong to a collaboration (big or small) nor to any particular campus (big or small).  
+<h2>The Open Science is Open to You!</h2>
```

Comment:
This heading makes no grammatical sense to me. Did you mean “The Open Science _Pool_”?

Reply:
lol. yup!

Reply:
fixed